### DISCUSSION TITLE
Proposal: Add custom or optional tag to transcript format

### DISCUSSION THREAD

**Discussion by paulthemagno** [Podcastindex-org/podcast-namespace#459] [Enhancement Proposal]
I was looking to the JSON transcript format [here mentioned](https://github.com/Podcastindex-org/podcast-namespace/blob/main/transcripts/example.json).

```json
{
  "version": "1.0.0",
  "segments": [
    {
      "speaker": "Darth Vader",
      "startTime": 0.5,
      "endTime": 0.75,
      "body": "I"
    },
    {
      "speaker": "Darth Vader",
      "startTime": 1,
      "endTime": 1.25,
      "body": "am"
    },
    {
      "speaker": "Darth Vader",
      "startTime": 1.5,
      "endTime": 2.0,
      "body": "your"
    },
    {
      "speaker": "Darth Vader",
      "startTime": 2.25,
      "endTime": 2.50,
      "body": "father."
    },
    {
      "speaker": "Luke",
      "startTime": 2.75,
      "endTime": 3.0,
      "body": "Nooooo"
    }
  ]
}
```

Is it possible to add **custom/optional** fields at the same level of `version`? Or maybe is there a sort of `other`/`notes` field where we can add miscellaneous information?

**Comment by paulthemagno** [comment]
Any update on that?

**Comment by theDanielJLewis** [comment]
What kind of information do you want to add?

**Comment by paulthemagno** [comment]
@theDanielJLewis The idea is to save a field to figure out where the transcript is coming from.
So the name could be `source` or similar and the value should be a string.

**Comment by theDanielJLewis** [comment]
You can probably add whatever additional fields you want. But what do you want that information to do, and how will it benefit anyone, especially the audience?

**Comment by jamescridland** [comment]
Hi, Paul. There is a proposal to add the quality/generator of transcript - whether it's entirely automated or if it has gone through some form of human editorial review - at #421 

(I complicated it by also adding opt-out requests. I should split the proposal into two.)

This benefits the listener by making it clear when a transcript is the product of automation, or whether it can be 100% relied on after going through a human being. I would expect this to be visible in the UX in some way. Is this what you would be wanting to do? (In my mind, this information should be in the signpost to the transcript - ie the link in the RSS - rather than in the transcript itself; you can't add that kind of metadata into SRT/VTT files as I understand it.)

I'm not sure of the use-case to explain whether the transcript came from WhisperAI/Rev/Descript though: very open to it, but let us know.
